src/main/java/model/data/CombinedDataTable.java

```diff
@@ -123,6 +123,17 @@ public void deleteNotFlagged() {
 		}
 	}
 
+	@Override
+	public DataTable getTable(String name) {
+		if (table.getName().equals(name)) {
```

Comment:
Ik snap dit niet helemaal, in de praktijk krijg je dan toch "eentabel.getTable("eentabelzijnnaam")?
lijkt me een soort cirkelredenering